openshift/worker.yaml

```diff
@@ -245,16 +253,36 @@ parameters:
   name: NAMESPACE
   required: true
   value: myproject
+- description: Number of Celery Workers to create
+  displayName: Celery Worker Replicas
+  name: CELERY_WORKER_REPLICAS
+  required: true
+  value: '1'
+- description: Initial amount of memory the Flower container will request.
+  displayName: Celery Worker Memory Request
+  name: CELERY_WORKER_MEMORY_REQUEST
+  required: true
+  value: 1Gi
```

Comment:
If I counted right, as it stands all our requests add up to 4 GB. I think we'll have problems rolling out new updates in this case. A new build or deploy will trigger and fail because we don't have resources to create a new pod (as the new pod gets created before deleting the old). We should lower all our requests to be a fairly low value, just enough to get the thing started, but use as much of our 8GB of `limit`as we can.

Reply:
Setting the initial memory request to 512Mi